decentraland/unity-explorer
# Chore: multiplayer debug profile movement

## What does this PR change?

Add possibility to profile remote profile movement state.

## How to test the changes?

1. Launch the explorer
2. Open Mulitplayer Movement debug tab
3. Click "Instantiate Self-Replica"
4. Click ""Debug profile" and see markers in the panel to be updated while you runnig around and self-replica copies your movement. 

Additionally, you can try with somebody presented in the room, if you know they wallet - just past it instead of "SelfReplica" in the "Debug profile" input field.

Keep in mind - it is a debug tool. Don't need to be polished

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
